cicd.md

```
@@ -0,0 +1,45 @@
+        -  Configure shared library for your Jenkins Server Navigate through Dashboard > Manage Jenkins > System, and add Global Trusted Pipeline Libraries. (Modern SCM)
+
+        ![Shared-library](images/shared_library.png)
+
+
+    - Configure Crendentials.
+        - Credentials that are requied during job execution. 
+        - e.g. DockerHub credentials for push and pull images.
+        
+        ![Shared-library](images/credentials.png)
+
+3. #### Create a Pipeline, execute Job and Cofigure Weebhook..
```

Markdownlint

Punctuation: '..'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)